presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5719,7 +5719,7 @@ public void testShowTablesFrom()
     @Test
     public void testShowTablesLike()
     {
-        MaterializedResult result = computeActual("SHOW TABLES LIKE 'or%'");
```

Comment:
why?

Reply:
I can't tell you ;)

Reply:
Some tests create tables. When testa are executed concurrently (class-level), then tables created by other tests may exist when this test runs.
It's much more likely that a test creates a table matching `or*` than matching `orde*`.

Reply:
To me `or*` or `orde*` does not matter as probably there might be other `orders` tables which match to both patterns. So it does not solve the issue.

Reply:
Not in a single schema.

Reply:
Then maybe you should fix the other test to not to use this schema or to use a different table name?